jqplot_example/jqplot_example.links.menu.yml

```diff
@@ -0,0 +1,28 @@
+jqplot.example:
+  title: jqplot Example
```

Comment:
We are going to add dynamic examples also, so I think menu should clearly say static/dynamic examples.